src/internal/Observable.ts

```diff
@@ -340,7 +340,7 @@ export class Observable<T> implements Subscribable<T> {
    * @method Symbol.observable
    * @return {Observable} this instance of the observable
    */
-  [Symbol_observable]() {
+  [observableTypeRef.symbol](): Subscribable<T> {
```

Comment:
An explicit type annotation is required here because with the inferred `Observable<T>` TS can't infer correctly the type using `ObservedValueOf` and it ends up being `unknown`. I think that is caused by the fact that `Observable<T>["subscribe"]` is overloaded and TS is only using the last declared overload when processing a conditional type - so it fails to match the correct signature.